### PR TITLE
[Backend][Updater] Renamed facultyCodes to organisationConstants and moved universityOrgId there

### PR DESCRIPTION
**Pull request by mattirasanen**
Renamed `facultyCodes` to `organisationConstants`. Added new env variable `ROOT_ORG_ID` to overwrite universityOrgId default value will be same `hy-university-root-id`

### CODE REVIEW

File: updater/sis-updater-worker/src/updater/updateMeta.js [hunk 1]
```diff
@@ -3,14 +3,13 @@ const _ = require('lodash')
 
 const { bulkCreate, selectFromByIdsOrderBy } = require('../db')
 const { Course, CourseProvider, CourseType, CreditType, Organization, StudyrightExtent } = require('../db/models')
+const { universityOrgId } = require('../../../../services/backend/config/organisationConstants')
```
Comment (rikurauhala) [inline updater/sis-updater-worker/src/updater/updateMeta.js:6]: Updater and backend run in separate containers so this is not going to work. Even though the different services are stored in a common monorepo, they do not currently share any code between them.

File: updater/sis-updater-worker/src/updater/updateStudents/attainments.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ const {
   validAttainmentTypes,
   customAttainmentTypes,
 } = require('../mapper')
-const { universityOrgId } = require('../updateMeta')
```
Comment (rikurauhala) [inline updater/sis-updater-worker/src/updater/updateStudents/attainments.js:14]: Same here.

File: updater/sis-updater-worker/src/updater/updateMeta.js (hunk 1 above)
Comment (mattirasanen) [reply updater/sis-updater-worker/src/updater/updateMeta.js:6]: Oh yea we need to think other solution for this

Comment (mattirasanen) [reply updater/sis-updater-worker/src/updater/updateMeta.js]: Added new env variable `ROOT_ORG_ID` to overwrite it. Could this work?

**Review by EssKayz — APPROVED** [review]
Looks FD approved

**Review by rikurauhala — APPROVED** [review]

**Pull request merged**